diff -ru phplayersmenu-3.2.0/layersmenu-horizontal-1.txt phplayersmenu-3.2.0.onclick-simple/layersmenu-horizontal-1.txt
--- phplayersmenu-3.2.0/layersmenu-horizontal-1.txt	2004-08-14 23:17:21.000000000 +0200
+++ phplayersmenu-3.2.0.onclick-simple/layersmenu-horizontal-1.txt	2004-08-17 09:41:47.843431960 +0200
@@ -1,4 +1,5 @@
-.|...GNU's Not Unix!|http://www.gnu.org/|The Free Software Foundation
+.|...GNU's Not Unix!||The Free Software Foundation
+..|...GNU's Not Unix!|http://www.gnu.org/|The Free Software Foundation
 .|Open Source
 ..|Portals
 ...|O. S. Devel. Network|http://www.osdn.com/
diff -ru phplayersmenu-3.2.0/lib/layersmenu.inc.php phplayersmenu-3.2.0.onclick-simple/lib/layersmenu.inc.php
--- phplayersmenu-3.2.0/lib/layersmenu.inc.php	2004-08-14 23:37:18.000000000 +0200
+++ phplayersmenu-3.2.0.onclick-simple/lib/layersmenu.inc.php	2004-08-17 09:39:43.226376632 +0200
@@ -547,9 +547,9 @@
 				continue;
 			}
 			if ($this->tree[$cnt]['not_a_leaf']) {
-				$this->tree[$cnt]['onmouseover'] = ' onmouseover="moveLayerX1(' . "'" . $this->tree[$cnt]['layer_label'] . "', '" . $menu_name . "') ; LMPopUp('" . $this->tree[$cnt]['layer_label'] . "'" . ', false);"';
+				$this->tree[$cnt]['onmouseover'] = ' onclick="moveLayerX1(' . "'" . $this->tree[$cnt]['layer_label'] . "', '" . $menu_name . "') ; LMPopUp('" . $this->tree[$cnt]['layer_label'] . "'" . ', false);"';
 			} else {
-				$this->tree[$cnt]['onmouseover'] = ' onmouseover="shutdown();"';
+				$this->tree[$cnt]['onmouseover'] = ' onclick="shutdown();"';
 			}
 			$t->setVar(array(
 				'menu_layer_label'	=> $menu_name . $this->tree[$cnt]['layer_label'],
@@ -709,9 +709,9 @@
 				continue;
 			}
 			if ($this->tree[$cnt]['not_a_leaf']) {
-				$this->tree[$cnt]['onmouseover'] = ' onmouseover="moveLayerX(' . "'" . $this->tree[$cnt]['layer_label'] . "') ; moveLayerY('" . $this->tree[$cnt]['layer_label'] . "') ; LMPopUp('" . $this->tree[$cnt]['layer_label'] . "'" . ', false);"';
+				$this->tree[$cnt]['onmouseover'] = ' onclick="moveLayerX(' . "'" . $this->tree[$cnt]['layer_label'] . "') ; moveLayerY('" . $this->tree[$cnt]['layer_label'] . "') ; LMPopUp('" . $this->tree[$cnt]['layer_label'] . "'" . ', false);"';
 			} else {
-				$this->tree[$cnt]['onmouseover'] = ' onmouseover="shutdown();"';
+				$this->tree[$cnt]['onmouseover'] = ' onclick="shutdown();"';
 			}
 			$t->setVar(array(
 				'imgwww'	=> $this->imgwww,
