diff -ru phplayersmenu-3.2.0/example-hormenu.php phplayersmenu-3.2.0.fixedmenu/example-hormenu.php
--- phplayersmenu-3.2.0/example-hormenu.php	2004-08-14 23:27:27.000000000 +0200
+++ phplayersmenu-3.2.0.fixedmenu/example-hormenu.php	2004-08-17 10:40:59.000000000 +0200
@@ -90,6 +90,14 @@
 */
 ?>
 
+<script language="JavaScript" type="text/javascript">
+<!--
+if (!IE) {
+	document.write('<div class="normal" style="height: 21px;">&nbsp;</div>');
+}
+// -->
+</script>
+
 <div class="normalbox">
 <div class="normal" align="center">
 <b><?php print basename(__FILE__); ?> - a file-based example with a Horizontal Layers Menu</b>
diff -ru phplayersmenu-3.2.0/layersmenu-gtk2.css phplayersmenu-3.2.0.fixedmenu/layersmenu-gtk2.css
--- phplayersmenu-3.2.0/layersmenu-gtk2.css	2004-08-14 16:39:02.000000000 +0200
+++ phplayersmenu-3.2.0.fixedmenu/layersmenu-gtk2.css	2004-08-17 10:50:51.000000000 +0200
@@ -2,6 +2,12 @@
 	color: black;
 	background-color: #dcdad5;
 	border: 2px outset #cecac1;
+/*	width: 100%; */
+/*	position: absolute; */
+/*	top: 0px; */
+/*	right: 0px; */
+/*	left: 0px; */
+	position: fixed;
 }
 
 .horbaritem	{
diff -ru phplayersmenu-3.2.0/libjs/layersmenu.js phplayersmenu-3.2.0.fixedmenu/libjs/layersmenu.js
--- phplayersmenu-3.2.0/libjs/layersmenu.js	2004-08-16 14:03:38.000000000 +0200
+++ phplayersmenu-3.2.0.fixedmenu/libjs/layersmenu.js	2004-08-17 10:37:34.000000000 +0200
@@ -179,6 +179,7 @@
 	}
 }
 	setLeft(menuName, foobar);
+	if (!IE) { setTop(menuName, getWindowYOffset() + getOffsetHeight('fixedmenu') - 1); }
 }
 
 function layersOverlap(layer, i)
diff -ru phplayersmenu-3.2.0/templates/layersmenu-horizontal_menu.ihtml phplayersmenu-3.2.0.fixedmenu/templates/layersmenu-horizontal_menu.ihtml
--- phplayersmenu-3.2.0/templates/layersmenu-horizontal_menu.ihtml	2004-07-04 10:59:58.000000000 +0200
+++ phplayersmenu-3.2.0.fixedmenu/templates/layersmenu-horizontal_menu.ihtml	2004-08-17 10:35:11.000000000 +0200
@@ -23,7 +23,7 @@
 <table border="0" cellspacing="0" cellpadding="0">
 <tr>
 <td>
-<div class="horbar">
+<div class="horbar" id="fixedmenu">
 <table border="0" cellspacing="0" cellpadding="0">
 <tr>
 <!-- BEGIN horizontal_menu_cell -->
