diff -ru phplayersmenu-3.2.0/DUMPS/mysql.demo_data.dump phplayersmenu-3.2.0.icons_i18n/DUMPS/mysql.demo_data.dump
--- phplayersmenu-3.2.0/DUMPS/mysql.demo_data.dump	2004-08-15 08:56:43.000000000 +0200
+++ phplayersmenu-3.2.0.icons_i18n/DUMPS/mysql.demo_data.dump	2004-08-17 11:58:23.811455432 +0200
@@ -57,6 +57,6 @@
 INSERT INTO phplayersmenu VALUES ('490', '370', 'The DOM Event Model', 'http://www.brainjar.com/dhtml/events/default4.asp', '', '', 'DOM', '710', '0');
 INSERT INTO phplayersmenu VALUES ('500', '370', 'Opera JS Support', 'http://www.opera.com/docs/specs/js/', '', '', 'DOM', '720', '0');
 
-INSERT INTO phplayersmenu_i18n VALUES ('it', '280', 'Navigatori', '');
-INSERT INTO phplayersmenu_i18n VALUES ('it', '330', 'Navigatori non liberi', '');
+INSERT INTO phplayersmenu_i18n VALUES ('it', '280', 'Navigatori', '', 'mozilla.org_images_mozilla-16.png');
+INSERT INTO phplayersmenu_i18n VALUES ('it', '330', 'Navigatori non liberi', '', '');
 
diff -ru phplayersmenu-3.2.0/DUMPS/mysql.start.dump phplayersmenu-3.2.0.icons_i18n/DUMPS/mysql.start.dump
--- phplayersmenu-3.2.0/DUMPS/mysql.start.dump	2004-07-29 21:29:32.000000000 +0200
+++ phplayersmenu-3.2.0.icons_i18n/DUMPS/mysql.start.dump	2004-08-17 11:57:29.760672400 +0200
@@ -26,6 +26,7 @@
 	id int NOT NULL,
 	text text,
 	title text,
+	icon text,
 	PRIMARY KEY (language, id)
 );
 
diff -ru phplayersmenu-3.2.0/DUMPS/pgsql.demo_data.dump phplayersmenu-3.2.0.icons_i18n/DUMPS/pgsql.demo_data.dump
--- phplayersmenu-3.2.0/DUMPS/pgsql.demo_data.dump	2004-08-15 08:57:12.000000000 +0200
+++ phplayersmenu-3.2.0.icons_i18n/DUMPS/pgsql.demo_data.dump	2004-08-17 11:59:53.882762512 +0200
@@ -57,6 +57,6 @@
 INSERT INTO phplayersmenu VALUES ('490', '370', 'The DOM Event Model', 'http://www.brainjar.com/dhtml/events/default4.asp', '', '', 'DOM', '710', '0');
 INSERT INTO phplayersmenu VALUES ('500', '370', 'Opera JS Support', 'http://www.opera.com/docs/specs/js/', '', '', 'DOM', '720', '0');
 
-INSERT INTO phplayersmenu_i18n VALUES ('it', '280', 'Navigatori', '');
-INSERT INTO phplayersmenu_i18n VALUES ('it', '330', 'Navigatori non liberi', '');
+INSERT INTO phplayersmenu_i18n VALUES ('it', '280', 'Navigatori', '', 'mozilla.org_images_mozilla-16.png');
+INSERT INTO phplayersmenu_i18n VALUES ('it', '330', 'Navigatori non liberi', '', '');
 
diff -ru phplayersmenu-3.2.0/DUMPS/pgsql.start.dump phplayersmenu-3.2.0.icons_i18n/DUMPS/pgsql.start.dump
--- phplayersmenu-3.2.0/DUMPS/pgsql.start.dump	2003-09-19 18:18:57.000000000 +0200
+++ phplayersmenu-3.2.0.icons_i18n/DUMPS/pgsql.start.dump	2004-08-17 12:00:13.289812192 +0200
@@ -25,6 +25,7 @@
 	id int2 NOT NULL,
 	text text,
 	title text,
+	icon text,
 	PRIMARY KEY (language, id)
 );
 
diff -ru phplayersmenu-3.2.0/DUMPS/sqlite.demo_data.dump phplayersmenu-3.2.0.icons_i18n/DUMPS/sqlite.demo_data.dump
--- phplayersmenu-3.2.0/DUMPS/sqlite.demo_data.dump	2004-08-15 08:58:05.000000000 +0200
+++ phplayersmenu-3.2.0.icons_i18n/DUMPS/sqlite.demo_data.dump	2004-08-17 11:58:07.691905976 +0200
@@ -50,6 +50,6 @@
 INSERT INTO phplayersmenu VALUES ('490', '370', 'The DOM Event Model', 'http://www.brainjar.com/dhtml/events/default4.asp', '', '', 'DOM', '710', '0');
 INSERT INTO phplayersmenu VALUES ('500', '370', 'Opera JS Support', 'http://www.opera.com/docs/specs/js/', '', '', 'DOM', '720', '0');
 
-INSERT INTO phplayersmenu_i18n VALUES ('it', '280', 'Navigatori', '');
-INSERT INTO phplayersmenu_i18n VALUES ('it', '330', 'Navigatori non liberi', '');
+INSERT INTO phplayersmenu_i18n VALUES ('it', '280', 'Navigatori', '', 'mozilla.org_images_mozilla-16.png');
+INSERT INTO phplayersmenu_i18n VALUES ('it', '330', 'Navigatori non liberi', '', '');
 
diff -ru phplayersmenu-3.2.0/DUMPS/sqlite.start.dump phplayersmenu-3.2.0.icons_i18n/DUMPS/sqlite.start.dump
--- phplayersmenu-3.2.0/DUMPS/sqlite.start.dump	2004-07-30 23:40:15.000000000 +0200
+++ phplayersmenu-3.2.0.icons_i18n/DUMPS/sqlite.start.dump	2004-08-17 11:59:23.097442592 +0200
@@ -16,6 +16,7 @@
 	id INTEGER NOT NULL,
 	text TEXT,
 	title TEXT,
+	icon text,
 	PRIMARY KEY (language, id)
 );
 
diff -ru phplayersmenu-3.2.0/example-db-hormenu.php phplayersmenu-3.2.0.icons_i18n/example-db-hormenu.php
--- phplayersmenu-3.2.0/example-db-hormenu.php	2004-08-15 08:59:46.000000000 +0200
+++ phplayersmenu-3.2.0.icons_i18n/example-db-hormenu.php	2004-08-17 12:01:48.915274912 +0200
@@ -75,7 +75,8 @@
 	'language'	=> 'language',
 	'id'		=> 'id',
 	'text'		=> 'text',
-	'title'		=> 'title'
+	'title'		=> 'title',
+	'icon'		=> 'icon'
 ));
 */
 /* HOWTO use data taken from the PgMarket 'categories'
@@ -100,7 +101,8 @@
 	'language'	=> 'lang',
 	'id'		=> 'category_id',
 	'text'		=> 'name',
-	'title'		=> ''
+	'title'		=> '',
+	'icon'		=> ''
 ));
 $mid->setPrependedUrl('/~pratesi/pgmarket/shopping/index.php?id=');
 //$mid->setIconsize(16, 16);
diff -ru phplayersmenu-3.2.0/lib/layersmenu-common.inc.php phplayersmenu-3.2.0.icons_i18n/lib/layersmenu-common.inc.php
--- phplayersmenu-3.2.0/lib/layersmenu-common.inc.php	2004-08-14 23:37:34.000000000 +0200
+++ phplayersmenu-3.2.0.icons_i18n/lib/layersmenu-common.inc.php	2004-08-17 12:04:50.716636904 +0200
@@ -217,7 +217,8 @@
 	'language'	=> 'language',
 	'id'		=> 'id',
 	'text'		=> 'text',
-	'title'		=> 'title'
+	'title'		=> 'title',
+	'icon'		=> 'icon'
 );
 /**
 * A temporary array to store data retrieved from the DB and to perform the depth-first search
@@ -708,7 +709,8 @@
 			SELECT ' .
 				$this->tableFields_i18n['id'] . ' AS id, ' .
 				$this->tableFields_i18n['text'] . ' AS text, ' .
-				$this->tableFields_i18n['title'] . ' AS title
+				$this->tableFields_i18n['title'] . ' AS title, ' .
+				$this->tableFields_i18n['icon'] . ' AS icon
 			FROM ' . $this->tableName_i18n . '
 			WHERE ' . $this->tableFields_i18n['id'] . ' <> 1
 				AND ' . $this->tableFields_i18n['language'] . ' = ' . "'$language'" . '
@@ -717,6 +719,7 @@
 			if (isset($this->_tmpArray[$row['id']])) {
 				$this->_tmpArray[$row['id']]['text'] = $row['text'];
 				$this->_tmpArray[$row['id']]['title'] = $row['title'];
+				$this->_tmpArray[$row['id']]['icon'] = $row['icon'];
 			}
 		}
 	}
